### ISSUE TITLE
마지막 학습단계에서...

### ISSUE THREAD

**Issue by Sun-Pak** [SKTBrain/KoBERT#45]
질문1)
마지막 에폭 돌리는 코드에서 
 out = model(token_ids, valid_length, segment_ids)
이부분에 에러가 발생합니다.

에러: dropout(): argument 'input' (position 1) must be Tensor, not str

질문2)
파이토치 설정 부분에서
>>> import torch
>>> from kobert.pytorch_kobert import get_pytorch_kobert_model
>>> input_ids = torch.LongTensor([[31, 51, 99], [15, 5, 0]])
>>> input_mask = torch.LongTensor([[1, 1, 1], [1, 1, 0]])
>>> token_type_ids = torch.LongTensor([[0, 0, 1], [0, 1, 0]])
>>> model, vocab  = get_pytorch_kobert_model()
>>> sequence_output, pooled_output = model(input_ids, input_mask, token_type_ids)
>>> pooled_output.shape
torch.Size([2, 768])
>>> vocab
Vocab(size=8002, unk="[UNK]", reserved="['[MASK]', '[SEP]', '[CLS]']")
>>> # Last Encoding Layer
>>> sequence_output[0]
tensor([[-0.2461,  0.2428,  0.2590,  ..., -0.4861, -0.0731,  0.0756],
        [-0.2478,  0.2420,  0.2552,  ..., -0.4877, -0.0727,  0.0754],
        [-0.2472,  0.2420,  0.2561,  ..., -0.4874, -0.0733,  0.0765]],
       grad_fn=<SelectBackward>)

이값은 코랩코드에는 나타나있지않은데,
코랩으로 돌릴때 설정해주어야 하는건가요?

**Comment by haven-jeon** [comment]
다시 설치 부탁드립니다.
